packages/wonder-blocks-form/util/types.js

```diff
@@ -22,6 +22,8 @@ export type ChoiceCoreProps = {|
      * guarantee that the ID is unique within everything rendered on a page.
      * Used to match <label> with <input> elements for screenreaders. */
     id?: string,
+    /** Optionally state that filling in or checking the input is required */
+    required?: boolean,
```

Comment:
For some reason, we are (re)defining these props in different locations (maybe for documentation purposes). e.g. https://github.com/Khan/wonder-blocks/blob/checkbox-required/packages/wonder-blocks-form/components/checkbox.js#L8

@kevinbarabash @jeresig Do you know why we are doing this?

Reply:
It was for documentation purposes because react-docgen doesn't understand spreads. :(

Reply:
`ChoiceCoreProps` is used in `CheckboxCore`, but not by `Checkbox`.  Since `Checkbox` and `Radio` are the publicly expose components their props will need to be updated as well.